metaworld/envs/mujoco/sawyer_xyz/v2/sawyer_stick_push_v2.py

```diff
@@ -109,59 +109,140 @@ def reset_model(self):
             while np.linalg.norm(goal_pos[:2] - goal_pos[-3:-1]) < 0.1:
                 goal_pos = self._get_state_rand_vec()
             self.stick_init_pos = np.concatenate((goal_pos[:2], [self.stick_init_pos[-1]]))
-            self._target_pos = np.concatenate((goal_pos[-3:-1], [self.stick_init_pos[-1]]))
+            self._target_pos = np.concatenate((goal_pos[-3:-1], [self._get_site_pos('insertion')[-1]]))
 
         self._set_stick_xyz(self.stick_init_pos)
         self._set_obj_xyz(self.obj_init_qpos)
         self.obj_init_pos = self.get_body_com('object').copy()
 
         return self._get_obs()
+    
+    def _gripper_caging_reward(self,
```

Comment:
What's the difference between this and the super() version?

Reply:
so this environment uses `self.stick_init_pos` to represent the initial stick position and `self.obj_init_pos` to represent the initial location of the mug. Since we are picking up the mug, I had to make those changes to the gripper reward function in order for it to work. I want to rename all the instances of stick location and mug location (which is always called `obj`) in both the environment python and xml files, but in order to be consistent with the old version I have left those values the same for now.